android/assets/jsons/translations/German.properties

```diff
@@ -3194,90 +3174,48 @@ Borgholm = Borgholm
- # Requires translation!
-Oh great queen, bold and dignified, the time has come for you to rise and guide the kingdom once again. Can you return your people to the height of prosperity and splendor? Will you build a civilization that stands the test of time? = 
- # Requires translation!
-Vienna = 
- # Requires translation!
-Salzburg = 
- # Requires translation!
-Graz = 
- # Requires translation!
-Linz = 
- # Requires translation!
-Klagenfurt = 
- # Requires translation!
-Bregenz = 
- # Requires translation!
-Innsbruck = 
- # Requires translation!
-Kitzbühel = 
- # Requires translation!
-St. Pölten = 
- # Requires translation!
-Eisenstadt = 
- # Requires translation!
-Villach = 
- # Requires translation!
-Zwettl = 
- # Requires translation!
-Traun = 
- # Requires translation!
-Wels = 
- # Requires translation!
-Dornbirn = 
- # Requires translation!
-Feldkirch = 
- # Requires translation!
-Amstetten = 
- # Requires translation!
-Bad Ischl = 
- # Requires translation!
-Wolfsberg = 
- # Requires translation!
-Kufstein = 
- # Requires translation!
-Leoben = 
- # Requires translation!
-Klosterneuburg = 
- # Requires translation!
-Leonding = 
- # Requires translation!
-Kapfenberg = 
- # Requires translation!
-Hallein = 
- # Requires translation!
-Bischofshofen = 
- # Requires translation!
-Waidhofen = 
- # Requires translation!
-Saalbach = 
- # Requires translation!
-Lienz = 
- # Requires translation!
-Steyr = 
+Austria = Österreich
+Maria Theresa = Maria Theresa
+Shame that it has come this far. But ye wished it so. Next time, be so good, choose your words more wisely.  = Schade, dass es so weit gekommen ist. Aber du hast es so gewollt. Sei das nächste Mal so gut, wähle Deine Worte mit Bedacht.
+What a fool ye are! Ye will end swiftly and miserably. = Was für ein Dummkopf seid ihr! Ihr werdet schnell und elend enden.
+The world is pitiful! There's no beauty in it, no wisdom. I am almost glad to go. = Die Welt ist erbärmlich! Darin liegt keine Schönheit, keine Weisheit. Ich bin fast froh zu gehen.
+The archduchess of Austria welcomes your Eminence to... Oh let's get this over with! I have a luncheon at four o'clock. = Die Erzherzogin von Österreich heißt Eure Eminenz herzlich willkommen... Ach, lasst uns das hinter uns bringen! Ich esse um vier Uhr zu Mittag.
+I see you admire my new damask. Nobody should say that I am an unjust woman. Let's reach an agreement! = I see you admire my new damask. Nobody should say that I am an unjust woman. Let's reach an agreement!
```

Comment:
Hier fehlt die Übersetzung gänzlich.

Reply:
There's a transcript of the voice acted lines [here](https://civilization.fandom.com/wiki/Maria_Theresa_(Civ5)), why not use those? Unfortunately the greeting "Hello!" is used elsewhere so we can't translate it to "Grüss Gott!"

Reply:
Thats a good @SimonCeder. I didn't know that the lines are already translated in the wiki.